src/partials/header.html

```diff
@@ -0,0 +1,35 @@
+<header class="header__line">
+    <a class="logo" href="index.html" title="Main page">
+        <svg class="logo__icon" width="35.28" height="42.8">
+            <use href="#"></use>
```

Comment:
use narazie robimy bez ścieżek ponieważ wywala to błędy w lokalnym serwerze, tyczy się to wszystkich use'ów